app/web/features/auth/locales/en.json

```diff
@@ -16,11 +16,13 @@
   },
   "change_password_form": {
     "title": "Change password",
+    "subtitle": "Please change password",
     "confirm_password": "Confirm password",
-    "password_changed_success": "Your new password has been set and a notification was sent to your email.",
-    "old_password": "Old password",
```

Comment:
These are used in `ChangePassword.tsx`? (When you change your password while logged in.)

Reply:
ups, reverted!